packages/generator-nitro/generators/app/templates/dash4.config.js

```diff
@@ -0,0 +1,90 @@
+/**
+ *
+ * 	DASH4 configuration
+ *  https://github.com/smollweide/dash4
+ *
+ */
+const { PluginTerminal } = require('@dash4/plugin-terminal');
+const { PluginReadme } = require('@dash4/plugin-readme');
+const { PluginNpmScripts } = require('@dash4/plugin-npm-scripts');
+const { PluginActions } = require('@dash4/plugin-actions');
+
+async function getConfig() {
+	return {
+		port: 4444,
+		tabs: [
+			{
+				title: 'START',
+				rows: [
+					[
+						new PluginReadme({ file: 'readme.md' }),
+					],
+					[
+						new PluginTerminal({
+							title: 'Development Mode',
+							cmd: 'npm run dev',
+							dark: true,
+							autostart: false,
+						}),
+						new PluginTerminal({
+							title: 'Production Mode',
+							cmd: 'npm run prod',
+							dark: true,
+							autostart: false,
+						}),
+						new PluginTerminal({
+							title: 'Cypress Mode',
+							cmd: 'npm run cypress-test',
+							dark: true,
+							autostart: false,
+						}),
+					],
+					[
+						new PluginNpmScripts({
+							scripts: [
```

Comment:
Is it possible to define a group title like with `PluginActions`?

Reply:
no. 
What would you write down as a title?

Reply:
let's make a feature request